vllm/vision_models/app.py

```diff
@@ -0,0 +1,97 @@
+from beam import endpoint, Image
+
+MODEL_NAME = "OpenGVLab/InternVL2_5-8B"
+
+vllm_image = Image(python_version="python3.10", python_packages=["vllm==0.6.3.post1", "fastapi[standard]==0.115.4"])
+
+@endpoint(
+    name="internvl2_5-1b",
+    image=vllm_image,
+    cpu=4,
+    memory="32Gi",
+    gpu="A10G",
+    gpu_count=2,
+)
+def generate():
+    import fastapi
+    import vllm.entrypoints.openai.api_server as api_server
+    from vllm.engine.arg_utils import AsyncEngineArgs
+    from vllm.engine.async_llm_engine import AsyncLLMEngine
+    from vllm.entrypoints.logger import RequestLogger
+    from vllm.entrypoints.openai.serving_chat import OpenAIServingChat
+    from vllm.entrypoints.openai.serving_completion import (
+        OpenAIServingCompletion,
+    )
+    from vllm.entrypoints.openai.serving_engine import BaseModelPath
+    from vllm.usage.usage_lib import UsageContext
+
+    web_app = fastapi.FastAPI(
+        title=f"OpenAI-compatible server",
```

Comment:
can we use the beam vLLM wrapper instead?

https://docs.beam.cloud/v2/reference/sdk#vllm

Reply:
@mernit The model requires two A10Gs, and the VLLM class doesn't have a gpu_count option.

Reply:
And here (https://github.com/beam-cloud/beta9/blob/3dda0e3ca199c69f688ed5358874021ad0dd1cfc/sdk/src/beta9/abstractions/integrations/vllm.py#L46C5-L46C33), it seems like the `enable_prompt_tokens_details parameter` is not being utilized. Even though I pass this argument to the VLLMArgs class, it doesn't seem to have any effect.

 I get this error ```enable_prompt_tokens_details=args.enable_prompt_tokens_details, AttributeError: 'types.SimpleNamespace' object has no attribute 'enable_prompt_tokens_details'```